plasmapy/tests/__init__.py

```diff
@@ -1,3 +1,5 @@
 """PlasmaPy tests and test helpers."""
 
+__all__ = ["helpers"]  # is this needed?
```

Comment:
Probably not really. https://plasmapy--923.org.readthedocs.build/en/923/api_static/plasmapy.tests.html isn't a thing, and https://plasmapy--923.org.readthedocs.build/en/923/api_static/plasmapy.tests.helpers.html#module-plasmapy.tests.helpers probably doesn't care about this `__all__` statement.
```suggestion
```